man/systemd.network.xml

```diff
@@ -1279,6 +1285,17 @@ DHCP=yes
 </programlisting>
     </example>
 
+    <example>
+      <title>/etc/systemd/network/25-vrf.network</title>
+
+      <programlisting>[Match]
+Name=bond1
+
+[Network]
+Vrf=vrf-test
```

Comment:
Can you add an explanation what this example means. Something like "\<para\>Add the bond1 interface to the VRF master interface vrf-test. This will cause packets ...\</para\>".